lib/node/runner/checker.go

```diff
@@ -306,7 +306,10 @@ func getMissingTransaction(checker *BallotChecker) (err error) {
 	}
 
 	for _, tx := range receivedTransaction {
-		checker.NodeRunner.TransactionPool.Add(tx)
+		err := checker.NodeRunner.TransactionPool.Add(tx)
+		if err == errors.ErrorTransactionPoolFull {
+			return err
+		}
```

Comment:
Won't that be problematic though ? I get the need to limit the tx pool, and it makes sense to reject client transactions, but not being able to add missing transactions could mean we're forever stuck, no ?

Reply:
I think this discuss should make another pr

Reply:
@anarcher : Not really, because it would mean we scrap most of this PR.

Reply:
Ok l will close it.

Reply:
@Geod24 Plz make a issue about that

Reply:
@Geod24 Plz make a issue about that

Reply:
I added a comment in #588